lib/linter-coffee-variables.coffee

```diff
@@ -1,9 +1,12 @@
 debug = require './debug'
+LRU = require('lru')
+coffee = require 'coffee-script'
 
-cache = {}
+cache = new LRU
+  max: 10
+  maxAge: 60 * 60 * 1000
 
 _compileToJS = (coffeeSource) ->
-  coffee = require 'coffee-script'
```

Comment:
As weird as this is, I originally did it to attempt to cut down the time the plugin took to initialize, which slowed down Atom's initial load. But I'm willing to put it back to the top of the file and see if it really makes a difference.